3648/2/commands/checkin.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'help_methods'
+require_relative 'download'
+require 'date'
+
+module CheckinCommand
+  include HelpMethods
+  include DownloadHelper
+  def checkin!(*)
+    if registred?
+      save_context :get_photo
+      respond_with :message, text: 'Отправьте фото'
+    else
+      respond_with :message, text: 'Вы не вошли'
+    end
+  end
+
+  def get_photo(_context = nil, *)
+    if photo?
+      timestamp_checkin
+      save_context :get_geo
+      respond_with :message, text: 'Гео'
```

Comment:
Try to write clear messages for users. Just 'Geo' don't give enough information, what user should do